includes/class-wc-product-variable.php

```diff
@@ -16,27 +16,25 @@
 class WC_Product_Variable extends WC_Product {
 
 	/**
-	 * Stores product data.
+	 * Array of children variation IDs. Determined by children.
 	 *
 	 * @var array
 	 */
-	protected $extra_data = array(
```

Comment:
@franticpsyx They are not set via CRUD, they are also read only. So I didn't want to reveal them when using get_data()